frontend/src/router/index.js

```diff
@@ -24,11 +23,6 @@ export default new Router({
     scrollBehavior: () => ({y: 0}),
     routes: [
         ...middleware(middlewares.auth(store))([
-            {
-                path: '/',
-                name: 'HelloWorld',
-                component: HelloWorld,
-            },
```

Comment:
do not remove path '/' just remove the link to HelloWorld

Reply:
I understand, that we'll need `/` route.
At that moment I didn't have a component to point root path at, so I decided to remove all route's part. I thought that next participant can add right component with root path `/`.
If it's a problem, I'm return it block now.